pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -313,3 +313,16 @@ def test_order_aggregate_multiple_funcs():
     expected = pd.Index(['sum', 'max', 'mean', 'ohlc', 'min'])
 
     tm.assert_index_equal(result, expected)
+
+
+@pytest.mark.parametrize('dtype', [np.int64, np.uint64])
+@pytest.mark.parametrize('how', ['first', 'last', 'min',
+                                 'max', 'mean', 'median'])
+def test_uint64_type_handling(dtype, how):
+    # GH 26310
+    df = pd.DataFrame({'x': 6903052872240755750, 'y': [1, 2]})
+    expected = df.groupby('y').agg({'x': how})
+    df.x = df.x.astype(dtype)
+    result = df.groupby('y').agg({'x': how})
+    result.x = result.x.astype(np.int64)
```

Comment:
Can we add test coverage for a value that exceeds the upper limit of an int64?

Reply:
It's not immediately clear to me what we would be testing for in that case. Issue #26310 is basically that there is some method `f` such that `f(x) != f(y)`, even though `x == y`, when `x` is of type `np.int64` and `y` is of type `np.uint64`.

If the value of `y` would exceed the upper limit of `np.int64` you couldn't represent it as an `np.int64` (?) and I'm not sure how you could pick `x` in that case to produce the above situation.

If I have misunderstood, please elaborate.

Reply:
The problem here is going to be with unsigned integers in the range of 2**63 through 2**64-1 (range where uint64 exceeds int64). I'm not sure if that would even work with these agg functions and don't want to open up a Pandora's box here but if that doesn't work we might just need to reword the whatsnew

Reply:
I suggest rewording whatsnew here and dealing with that in another pull request. What do you think @WillAyd? How would you like the whatsnew to be worded?

Reply:
hmm, input uint64's *should* work with your code change, but @mahepe ok to make an issue about this.